styles/main.css

```diff
@@ -0,0 +1,123 @@
+/* START header */
+
+.red-main-header {
+  background-color: #f72210;
+  fill: white;
+  color: white;
+}
+
+.bg-hoover-gray:hover {
+  background-color: #dbdcdd;
+}
+
+#input-sear-top-nav {
+  width: 28%;
+  outline: none;
+}
+
+.hover-red:hover {
+  color: #f72210 !important;
+}
```

Comment:
Please do not use !important rule so often. See read <a href = 'https://speckyboy.com/good-bad-css-practices/'>this article</a> for further reference.